### ISSUE TITLE
Ability to add label to list

### ISSUE THREAD

**Issue by bretto36** [Nanakii/summernote-plugins#6]
Would it be possible to add a label to the list value

something like 

```
list: {
    'invoice.name' : 'Invoice Name',
    'invoice.number' : 'Invoice Number',
    'invoice.date' : 'Invoice Date',
    'invoice.due_date' : 'Invoice Due Date'
}
```

**Comment by celestialaly** [comment]
with the key being the html file name ? I will look into it 😄

**Comment by bretto36** [comment]
Yep that was my thoughts. invoice.name will translate to invoice.name.html in the folder specified in config. 

Thanks!

**Comment by celestialaly** [comment]
I've implemented your suggestion in 125e5421965092b12f69e95e1e14eabe108aa013 :smile_cat: 

Thanks for the idea !
